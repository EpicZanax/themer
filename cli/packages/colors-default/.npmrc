tag-version-prefix "@themerdev/colors-default-v"
message "@themerdev/colors-default-v%s"
