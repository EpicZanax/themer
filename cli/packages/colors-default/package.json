{
  "name": "@themerdev/colors-default",
  "version": "2.0.0",
  "description": "A default color set for themer.",
  "main": "lib/index.mjs",
  "engines": {
    "node": ">=18"
  },
  "author": "mjswensen",
  "repository": {
    "url": "git+ssh://git@github.com/themerdev/themer.git",
    "type": "git",
    "directory": "cli/packages/colors-default"
  },
  "license": "MIT",
  "files": [
    "/lib/index.mjs"
  ],
  "scripts": {
    "prepublishOnly": "cp ../../../LICENSE.md ./",
    "test": "vitest"
  },
  "bugs": {
    "url": "https://github.com/themerdev/themer/issues"
  },
  "homepage": "https://github.com/themerdev/themer/tree/main/cli/packages/colors-default#readme",
  "keywords": [
    "themer",
    "colors",
    "colorset",
    "theme"
  ],
  "devDependencies": {
    "vitest": "^0.25.1"
  }
}
