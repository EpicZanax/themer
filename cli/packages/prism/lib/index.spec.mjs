import { render, renderInstructions } from './index.mjs';
import { colors } from '@themerdev/colors-default';
import { describe, expect, it } from 'vitest';

describe('render', () => {
  const promisedFiles = Promise.all(render(colors));
  it('should render the expected number of files', async () => {
    const files = await promisedFiles;
    expect(files.length).toBe(2);
  });
});

describe('renderInstructions', () => {
  it('should provide installation instructions', async () => {
    const files = await Promise.all(render(colors));
    const instructions = renderInstructions(files.map(({ name }) => name));
    expect(instructions).toMatchSnapshot();
  });
});
