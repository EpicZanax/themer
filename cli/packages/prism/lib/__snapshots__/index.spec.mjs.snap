// Vitest Snapshot v1

exports[`renderInstructions > should provide installation instructions 1`] = `
"
In the HTML page where prism.js is being used, link to the generated stylesheet (\`themer-prism-dark.css\` or \`themer-prism-light.css\`).
"
`;
